test/src/mbgl/test/sqlite3_test_fs.cpp

```diff
@@ -234,16 +234,21 @@ static int sqlite3_test_fs_access(sqlite3_vfs* vfs, const char* zPath, int flags
     if (!sqlite3_test_fs_io) {
         return SQLITE_AUTH;
     }
-    auto* unix_fs = (sqlite3_vfs*)vfs->pAppData;
-    return unix_fs->xAccess(unix_fs, zPath, flags, pResOut);
+    auto* os_fs = (sqlite3_vfs*)vfs->pAppData;
+    return os_fs->xAccess(os_fs, zPath, flags, pResOut);
 }
 
 namespace mbgl {
 namespace test {
 
 SQLite3TestFS::SQLite3TestFS() {
-    sqlite3_vfs* unix_fs = sqlite3_vfs_find("unix");
-    if (unix_fs == nullptr) {
+#ifdef WIN32
+    sqlite3_vfs* os_fs = sqlite3_vfs_find("win32");
```

Comment:
Maybe try passing `nullptr` here?

> If zVfsName is NULL then the default VFS is returned.

https://www.sqlite.org/c3ref/vfs_find.html

Reply:
Yes, much better this way! Correcting now.